packages/core/src/prompts/prompt.ts

```diff
@@ -145,8 +145,24 @@ export default class Prompt {
       this.state = 'active';
     }
 
-    if (key?.name && keys.has(key.name)) {
-      this.emit('cursor', key.name);
+    switch (key?.name) {
+      case 'h':
```

Comment:
What do you think about adding `h, j, k, l` to `keys` and instead convert them to the corresponding cursor movement before emitting the event?

Reply:
Well pointed, thank you